cli/rollup.config.js

```diff
@@ -81,7 +81,13 @@ export default [
     },
     plugins: [
       ...plugins,
-      execute(`chmod +x dist/${cliConfig.name}`),
+      {
+        name: 'closeBundle',
+        writeBundle: () => {
+          console.info(`make ${cliConfig.name} executable`);
```

Comment:
Get rid of this logging?

Reply:
I left this in intentionally. `rollup` logs the rest of the steps. Happy to remove it though.

Reply:
Let's leave in then.